spring-cloud-task-core/pom.xml

```diff
@@ -19,6 +19,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-autoconfigure</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-deployer-autoconfigure</artifactId>
```

Comment:
Why is this needed in core?  I'd expect this to be needed in the stream module...

Reply:
Yeah, I wasn't sure which one needed this and just picked core as it already have deps for `spring-cloud-deployer-resource-support` and `spring-cloud-deployer-resource-maven`.

Reply:
It is needed in both the stream and batch module.  The batch uses it for partitioning.

Reply:
ah, I think that `ObjectProvider.orderedStream()` were added to framework 5.1.x. I suppose if task builds on boot21/framework51, you would not expect boot20 apps be fully supported.

Reply:
task autoconfig is currently in `spring-cloud-task-core` so technically it's in a same place/level now with this pr.

Reply:
My bad on the first test.   I got it working.
LGTM